introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Will and I am excited to learn more about Git. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.